.travis.yml

```diff
@@ -9,7 +9,7 @@ bundler_args: --without development
 cache: bundler
 
 before_install:
-  - source script/install_phantomjs.sh
+  - npm install -g phantomjs-prebuilt
```

Comment:
I guess... http://stackoverflow.com/questions/30240237/how-do-i-cache-global-npm-packages-on-travis-ci. Let's try it.